ext/semian/sysv_semaphores.c

```diff
@@ -54,26 +70,6 @@ set_semaphore_permissions(int sem_id, long permissions)
 }
 
 int
-create_semaphore(int key, long permissions, int *created)
-{
-  int semid = 0;
-  int flags = 0;
-
-  *created = 0;
-  flags = IPC_EXCL | IPC_CREAT | permissions;
-
-  semid = semget(key, SI_NUM_SEMAPHORES, flags);
```

Comment:
note that this is essentially the example of the [bad way to do it](http://man7.org/tlpi/code/online/diff/svsem/svsem_bad_init.c.html)